templates/js/custom-copilot-basic/src/app/app.js.tpl

```diff
@@ -0,0 +1,41 @@
+const { MemoryStorage } = require("botbuilder");
+const path = require("path");
+const config = require("../config");
+
+// See https://aka.ms/teams-ai-library to learn more about the Teams AI library.
+const { Application, ActionPlanner, OpenAIModel, PromptManager } = require("@microsoft/teams-ai");
+
+// Create AI components
+const model = new OpenAIModel({
+  {{#useOpenAI}}
+  apiKey: config.openAIKey,
+  defaultModel: "gpt-3.5-turbo",
+  {{/useOpenAI}}
+  {{#useAzureOpenAI}}
+  azureApiKey: config.azureOpenAIKey,
+  azureDefaultDeployment: "gpt-35-turbo",
```

Comment:
How about add a comment to tell user don't forget to  update it himself

Reply:
The hint is included in readme file.